clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroStoreClient.java

```diff
@@ -537,14 +537,11 @@ public static <T, K> T tryToDeserializeWithVerboseLogging(
       }
 
       LOGGER.error(
-          "Caught a {}, will bubble up."
-              + "RecordDeserializer: {}\nWriter schema ID: {}\nLatest schema ID: {}\nValue (md5/hex): {}\nKey (hex): {}",
+          "Caught a {}, will bubble up." + "RecordDeserializer: {}\nWriter schema ID: {}\nLatest schema ID: {}\n",
           VeniceSerializationException.class.getSimpleName(),
           dataDeserializer.getClass().getSimpleName(),
           (writerSchemaId == -1 ? "N/A" : writerSchemaId),
-          latestSchemaId,
-          checksumHex,
-          keyHex);
```

Comment:
When I investigated a bit, these variables were computed just for the purpose of logging. Can we remove that too?

Reply:
They are computed to validate if they can be desered or not.